cluster/k8s-1.13.3/.kubeconfig

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
should not  be commited. Isn't .kubeconfig a part of gitignore?

Reply:
it is there is a problem with the commits on this PR I think he should rebase